### PR TITLE
Allow mapping org auditors where we map org admins.

### PR DESCRIPTION
**Pull request by wenottingham**
##### SUMMARY

To match system-level admin and auditor mapping.

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
 - API

### CODE REVIEW

**Comment by wenottingham** [comment]
cc @one-t @chrismeyersfsu

**Comment by wenottingham** [comment]
(see a comment on #576, this is obviously not *all* roles, but this is a useful one.)

**Comment by wenottingham** [comment]
recheck

**Comment by wenottingham** [comment]
recheck

**Comment by wenottingham** [comment]
recheck

**Comment by ryanpetrello** [comment]
This is a `priority:low` and `type:enhancements`, let's please hold off on merging this for now unless somebody has cycles to track and verify it.

**Review by ryanpetrello — DISMISSED** [review]
This is a `priority:low` and `type:enhancement`, let's please hold off on merging this for now unless somebody has cycles to track and verify it.

**Comment by wenottingham** [comment]
recheck

File: awx/sso/conf.py
Comment (chrismeyersfsu) [inline awx/sso/conf.py:56]: :+1:

**Review by chrismeyersfsu — APPROVED** [review]
On the surface, looks good. Would need QE to write integration tests to feel safe.

**Comment by wenottingham** [comment]
OK, @one-t approved the tower-qa tests, will merge when this merges.

**Comment by wenottingham** [comment]
regate

**Pull request merged**
